integration-tests/smoke/auto_ocr_test.go

```diff
@@ -173,6 +227,471 @@ var _ = Describe("Automation OCR Suite @auto-ocr", func() {
+			var countersAfterPause = make([]*big.Int, len(upkeepIDs))
+			for i := 0; i < len(upkeepIDs); i++ {
+				countersAfterPause[i], err = consumers[i].Counter(context.Background())
+				Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+					" for upkeep at index "+strconv.Itoa(i))
+			}
+
+			// After we paused the registry, the counters of all the upkeeps should stay constant
+			// because they are no longer getting serviced
+			Consistently(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					latestCounter, err := consumers[i].Counter(context.Background())
+					Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(latestCounter.Int64()).Should(Equal(countersAfterPause[i].Int64()),
+						"Expected consumer counter to remain constant at %d, but got %d",
+						countersAfterPause[i].Int64(), latestCounter.Int64())
+				}
+			}, "1m", "1s").Should(Succeed())
+		}
+
+		if testToRun == HandleKeeperNodesGoingDown {
+			By("upkeeps are still performed if some keeper nodes go down")
+			var initialCounters = make([]*big.Int, len(upkeepIDs))
+
+			// Watch upkeeps being performed and store their counters in order to compare them later in the test
+			Eventually(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					counter, err := consumers[i].Counter(context.Background())
+					initialCounters[i] = counter
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(counter.Int64()).Should(BeNumerically(">", int64(0)),
+						"Expected consumer counter to be greater than 0, but got %d", counter.Int64())
+				}
+			}, "5m", "1s").Should(Succeed())
+
+			// OCR change to take just f nodes first in this case f=1
+			fNodesDown := nodesWithoutBootstrap[0:1]
+			for _, nodeToTakeDown := range fNodesDown {
+				err = nodeToTakeDown.MustDeleteJob("1")
+				Expect(err).ShouldNot(HaveOccurred(), "Could not delete the job from one of the nodes")
+				err = chainClient.WaitForEvents()
+				Expect(err).ShouldNot(HaveOccurred(), "Error deleting the Keeper job from the node")
+			}
+			log.Info().Msg("Successfully managed to take down the first half of the nodes")
+
+			// Assert that upkeeps are still performed and their counters have increased
+			Eventually(func(g Gomega) {
+				for i := 0; i < len(upkeepIDs); i++ {
+					currentCounter, err := consumers[i].Counter(context.Background())
+					g.Expect(err).ShouldNot(HaveOccurred(), "Failed to retrieve consumer counter"+
+						" for upkeep at index "+strconv.Itoa(i))
+					g.Expect(currentCounter.Int64()).Should(BeNumerically(">", initialCounters[i].Int64()),
+						"Expected counter to have increased from initial value of %s, but got %s",
+						initialCounters[i], currentCounter)
+				}
+			}, "5m", "1s").Should(Succeed())
```

Comment:
same q here. How much time does it take usually in your run?

Reply:
i would imagine this will take longer than the previous run bc all the nodes need to digest the new config setup and restart. maybe try 2m and 4m?

Reply:
i see. I think that can be fixed by adding `contractConfigTrackerPollInterval = "1s"` in bootstrap and ocr job specs

Reply:
we can not set it that low anymore because they switched the testing nodes to not be in dev mode anymore. I will try lowering it and see how it goes.